### PR TITLE
Check XML files against schema

### PR DESCRIPTION
**Pull request by szepeviktor**
## PR Description

@dereuromark Hello! This is XML checking for you :)

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Comment by szepeviktor** [comment]
CI should fail as **there are** schema validation errors in XML ruleset file.

**Comment by szepeviktor** [comment]
💡 This PR needs #332

**Comment by szepeviktor** [comment]
@dereuromark I leave this 100% up to you.

I'm on GitHub to realize my dreams. This is tough business, not my place.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -97,3 +97,22 @@ jobs:
...
         run: vendor/bin/validate-prefer-lowest
+
+jobs:
+  xml-validation:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install xmllint tool
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libxml2-utils
```
Comment (szepeviktor) [inline .github/workflows/ci.yml:108]: Please see https://github.com/actions/virtual-environments/issues/423

**Review by dereuromark — APPROVED** [review]

**Pull request merged**